db/apply.go

```diff
@@ -234,6 +246,67 @@ func (chain *Chain) ApplyFAT0Tx(ei int64, e factom.Entry) (tx *fat0.Transaction,
 	return
 }
 
+var tmp grader.BlockGrader
```

Comment:
> this global is very very suspicious to me. Why can this not be created on the stack as its needed by functions?

it can be made a chain struct variable once there is a dedicated pegnet chain